README.md

```diff
@@ -104,11 +104,11 @@ Algunos recursos poseen un emoticón para ayudar a entender el tipo de contenido
 <link rel="icon" type="image/png" href="https://example.com/favicon.png">
 ```
 
-> * 🛠 [Favicon Generator](https://www.favicon-generator.org/)
-> * 🛠 [RealFaviconGenerator](https://realfavicongenerator.net/)
-> * 📖 [Favicon Cheat Sheet](https://github.com/audreyr/favicon-cheat-sheet)
-> * 📖 [Favicons, Touch Icons, Tile Icons, etc. Which Do You Need? - CSS Tricks](https://css-tricks.com/favicon-quiz/)
-> * 📖 [PNG favicons - caniuse](https://caniuse.com/#feat=link-icon-png)
+> * 🛠 [Generador de Favicon](https://www.favicon-generator.org/) `(Fuente en Inglés)`
```

Comment:
Links that have the 🛠 icon translate as `(Herramienta en Inglés)`.

Reply:
So the other ones, like books and tools we keep it in English??? or Will we change it later too?

Reply:
For a complete translation its better to translate all titles, that way people that read this repo knows what links are about even if are in another language.
That way they can google the title to find information about the topic in Spanish if they can´t read English.

Thank you 😃

Reply:
So I gonna close this PR.

Thank you :smile: